tests/README.md

```diff
@@ -2,27 +2,47 @@
 
 ## Overview
 
-This project contains UI smoke tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
+This project contains smoke and integration UI tests for Busola. Copy your cluster's kubeconfig file to [`fixtures/kubeconfig.yaml`](fixtures/kubeconfig.yaml).
 
 ## Installation
 
 To install dependencies, run the `npm install` command.
 
 ## Development
 
-To run UI tests inside a container in the cluster using a Chrome browser, use this command:
+To run Cypress UI tests using a Chrome browser in headless mode,
+pointing to remote Busola cluster with default local.kyma.dev domain use this command:
 
 ```bash
-npm run test:cluster
+npm test
 ```
 
-To run UI tests on a cluster, use this command:
+Pointing to remote Busola cluster, use this command:
+
+```bash
+CYPRESS_DOMAIN={YOUR_DOMAIN} npm test
+```
+
+Pointing to local Busola instance, use this command:
+
+```bash
+npm run test:local
+```
+
+To open Cypress UI tests,
```

Comment:
Hightlight important phrases

To open Cypress UI tests,
pointing to the `remote Busola` cluster with default `local.kyma.dev` domain use this command: